app/notifications/rest.py

```diff
@@ -384,3 +385,12 @@ def send_notification(notification_type):
 
     statsd_client.incr('notifications.api.{}'.format(notification_type))
     return jsonify(data={"notification": {"id": notification_id}}), 201
+
+
+@notifications.route('/notifications/statistics')
+def get_notification_statistics_for_today():
+    today = date.today()
+    statistics = notifications_dao.dao_get_notification_statistics_for_day(day=today)
+
+    data, errors = notifications_statistics_schema.dump(statistics, many=True)
+    return jsonify(data=data, date=today.isoformat()), 200
```

Comment:
Is date really needed?
It is also part of the return of notifications_dao.dao_get_notification_statistics_for_day(day=today)